jsdoc/Home.md

```diff
@@ -0,0 +1,90 @@
+# Braintree Web Drop-in Reference <span>v@VERSION</span>
+
+<span class="rule"></span>
+
+* [Overview](#overview)
+* [Setup](#setup)
+* [Usage](#usage)
+  * [Accepting cards](#accepting-cards)
+  * [Accepting PayPal](#accepting-paypal)
+  * [Localization](#localization)
+  * [Events](#events)
+* [Browser support](#browser-support)
+* [Additional resources](#additional-resources)
+
+<span class="rule"></span>
+
+<a id="overview"></a>
+## Overview
+
+Drop-in is a pre-made payments UI for desktop and mobile browsers to be used with cards, PayPal, and PayPal credit. It can be used as a one-time guest checkout or to offer saved payment methods with a client token created using a [customer ID](https://developers.braintreepayments.com/guides/drop-in/javascript/v3#customer-id).
+
+Test our our [demo app](/) with one of our [test card numbers](https://developers.braintreepayments.com/reference/general/testing/#no-credit-card-errors) or a [sandbox PayPal account](https://developer.paypal.com/docs/classic/lifecycle/sb_create-accounts/).
+
+If you have any feedback or questions, create a [GitHub issue](https://github.com/braintree/braintree-web-drop-in/issues) or [contact Braintree support](https://developers.braintreepayments.com/forms/contact).
+
+<a id="setup"></a>
+## Setup
+
+The Drop-in source is available from our CDN, that you can include in your project as a script tag:
+
+```html
+<script src="https://js.braintreegateway.com/web/dropin/@VERSION/js/dropin.min.js"></script>
```

Comment:
I interpreted this section as "how to include drop-in in your code" but I could go either way.